### PR TITLE
Sniff for a blank line after PHP open tag.

### LINKED ISSUE #57: Enforce to leave a new line between the PHP opening tag and the namespace.
```
<?php
namespace Namespace;
```
and even
```
<?php namespace Namespace;
```
are being allowed to use.
It seems to me that enforcing to write the
```
<?php

namespace Namespace;
```
way would be a logical path.
First of all, if the license header is added, the standard already checks if it is separated from the opening tag and the namespace with the empty new lines. Furthermore, it is a good addition to improve code style consistency in the scope of a project. Lastly, it is the way Symfony team writes the code :)

### PR DESCRIPTION
**Pull request by emaslak**
Closes #57

### CODE REVIEW

**Comment by tautrimas** [comment]
Test are needed for QA.

File: Sniffs/WhiteSpace/PHPOpenTagSniff.php [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+
+
+use PHP_CodeSniffer_File;
```
Comment (ex13) [inline Sniffs/WhiteSpace/PHPOpenTagSniff.php]: License header should be alike with the one in other files, just with ONGR Team author tag. Also namespace should go after the license header.
And I think it's time to report a new bug, namespace and use statements have 4 blank lines between them.

File: Tests/Unit/WhiteSpace/PHPOpenTagSniffTest.php [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the ONGR package.
+ *
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
```
Comment (ex13) [inline Tests/Unit/WhiteSpace/PHPOpenTagSniffTest.php]: License header should be the same as in other ongr-strict-standard files.

File: Tests/Unit/WhiteSpace/PHPOpenTagSniffTest.phptest [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?php echo'foo'?>
+
+<?php
+echo 'bar'
```
Comment (ex13) [inline Tests/Unit/WhiteSpace/PHPOpenTagSniffTest.phptest]: Add a semicolon to both echo statements.

Comment (tautrimas) [inline Tests/Unit/WhiteSpace/PHPOpenTagSniffTest.phptest]: Strange, that it didn't catch it as error. Maybe it's for `<?= 'foobar' ?>. I would still prefer a semicolon. Bug report anyone?

File: Sniffs/WhiteSpace/PHPOpenTagSniff.php [hunk 4]
```diff
@@ -0,0 +1,54 @@
+<?php
+
+/**
+ * ONGR_Sniffs_WhiteSpace_PHPOpenTagSniff.
+ *
+ * PHP version 5
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Greg Sherwood <gsherwood@squiz.net>
```
Comment (ex13) [inline Sniffs/WhiteSpace/PHPOpenTagSniff.php]: Author tag can be ONGR Team <info@ongr.io>

**Pull request merged**